procedures/auxtel/reduction_procedures/Parabolic_Focus_Analysis.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>corrected with the user_applied focus offset (from the analysis above) as well as the value in the currently loaded configuration file (in ts_config_latiss) for the ATSpectrograph CSC.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Sounds way better. </p>